.github/workflows/kubernetes-deploy.yaml

```diff
@@ -53,6 +53,7 @@ jobs:
             --set gateway.application.ray.cpu=1 \
             --set gateway.application.limits.keepClusterOnComplete=false \
             .
+          sleep 120
```

Comment:
instead of a sleep, could we do a readiness check instead? something like : 

`kubectl wait  --for=condition=Ready "$SOMEPOD" --timeout 5m`

Reply:
It's better.